api/v1alpha3/condition_consts.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+
+// Conditions and condition Reasons for the VSphereVM object
+
+const (
+
+	// VMProvisionedCondition documents the status of the provisioning of a VSphereVM.
+	VMProvisionedCondition clusterv1.ConditionType = "VMProvisioned"
+
+	// CloningReason documents (Severity=Info) a VSphereVM currently executing the clone operation.
+	CloningReason = "Cloning"
```

Comment:
one question, do we want condition to be on going or in the past

Reply:
VMProvisining happens only one time at the beginning of the machine lifecycle, so IMO it is fine to use past